test/extensions/matching/input_matchers/cel_matcher/cel_matcher_test.h

```diff
@@ -0,0 +1,630 @@
+#pragma once
+
+namespace Envoy {
+namespace Extensions {
+namespace Matching {
+namespace InputMatchers {
+namespace CelMatcher {
+
+// Compiled CEL expression string: request.headers['authenticated_user'] == 'staging'
+inline constexpr char RequestHeadeCelExprString[] = R"pb(
```

Comment:
nit: it's more common to have constexpr absl::string_view in envoy. `inline` is also not necessary, compiler should infer it.

Reply:
also, typo in the name: Heade -> Header

Reply:
I think it's just ParseFromString that will take string_view shortly when we upgrade protobuf. Don't have a strong opinion here.